bin/rialto/node/src/overseer.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
If there are no changes in this file can we actually fetch it as part of `build.rs` from the github repository (if non-existent?)